mkt/webapps/tasks.py

```diff
@@ -715,6 +715,10 @@ def _get_trending(app_id):
         return {}
 
     def _score(week1, week3):
+        # If last week app installs are < 100, this app isn't trending.
+        if week1 < 100.0:
```

Comment:
I just made it up. We'll likely want to increase it as app installs increase. I modeled all this after AMO's "hotness" calculations which drives its "Up & Coming" add-ons: https://github.com/mozilla/olympia/blob/74364be6dce33d96220493706451951bdbeb8151/apps/addons/cron.py#L322

But I should make that a constant so it's easy to tweak.